lib/plugins/inspec-compliance/lib/inspec-compliance/cli.rb

```diff
@@ -1,15 +1,17 @@
 
+require 'inspec/dist'
+
 require_relative 'api'
 
 module InspecPlugins
   module Compliance
     class CLI < Inspec.plugin(2, :cli_command)
-      subcommand_desc 'compliance SUBCOMMAND', 'Chef Compliance commands'
+      subcommand_desc 'compliance SUBCOMMAND', "#{Inspec::Dist::COMPLIANCE_PRODUCT_NAME} commands"
```

Comment:
These start to feel heavy. You can `include Inspec::Dist` in this class (and others) and then drop the namespacing to make it a bit more readable.

Reply:
@artem-sidorenko Are you comfortable making these changes? I'd love to get this all merged. Thanks =)